pkg/pillar/cmd/conntrack/conntrack.go

```diff
@@ -8,12 +8,50 @@ import (
 	"fmt"
 	"github.com/eriknordmark/netlink"
 	log "github.com/sirupsen/logrus"
+	"net"
 	"syscall"
 )
 
 func Run() {
 	// XXX curpartPtr := flag.String("c", "", "Current partition")
+	delFlow := flag.Bool("D", false, "Delete flow")
+	delSrcIP := flag.String("s", "", "Delete flow with Srouce IP")
+	delProto := flag.Int("p", 0, "Delete flow with protocol ID")
+	delFamily := flag.String("f", "", "Delete flow with ipv6")
+	delPort := flag.Int("P", 0, "Delete flow with port number")
 	flag.Parse()
+
+	// conntrack [-D <-s address> [-p proto]]
+	if *delFlow {
```

Comment:
conntrack-tools allows filtering flows based on marking value also. Can we do that here also?

Reply:
It's possible, but I think the goal here is the flush the flows which are not covered by the ACL yet(so normally we don't know what their mark will be), as my example yesterday that ICMP first had this 0xfffff, but the ACL to allow that is the mark of 0x1000104. If you think there is a usage to that, sure.

Reply:
When we have changed to a rule it would make sense to be able to flush the conntrack state which matches that marking value and/or the drop marking value. So for experimentation with that I think we should add the deletion based on the marking value.

Reply:
Note need to resolve the conflicts. The master is updated with the latest eriknordmark/netlink code

Reply:
I removed my changes in the netlink side. Although the latest lf-edge/eve, if I do a 'make proto-vendor', it gives me error.

Reply:
I still see go.mod and vendor diffs in this PR, plus it claims that it can't be rebased due to the conflicts.

I'd suggest a 
 git fetch upstream master
 git rebase -i upstream/master
and as part of that rebase remove the commits which only affect go.mod and vendor.